doc/system_requirements.md

```diff
@@ -193,6 +193,21 @@ Covers:
 
 Needs: dsn
 
+#### Stopping and Restarting the Database
```

Comment:
```suggestion
#### Stopping and Starting the Database
```

Reply:
Similarly, requirements should be renamed.

`req-stopping-and-starting-the-database~1`.